examples/vector_fe/vector_fe_ex7/vector_fe_ex7.C

```diff
@@ -15,6 +15,87 @@
 // License along with this library; if not, write to the Free Software
 // Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 
+
+// <h1>Vector Finite Elements Example 7 - Hybridized Raviart-Thomas elements</h1>
+// \author Alexander Lindsay
+// \date 2023
+//
+// This example hybridizes Raviart-Thomas elements to solve a model div-grad
+// problem in both 2d and 3d. Before hybridization, the mixed problem is simply
+// a div-grad formulation, \vec{u} = -\nabla p, and \nabla \cdot \vec{u} = f, of
+// the Poisson problem in Introduction Example 3, \nabla^2 p = -f. A standard
+// (non-hybridized) Raviart-Thomas discretization of the same problem can be
+// found in Vector Example 6.
+//
+// One of the first references for the hybridized Raviart-Thomas (RT)
+// formulation is Arnold's and Brezzi's "Mixed and nonconforming finite element
+// methods: implementation, postprocessing and error estimates". The key piece
+// to the hybridized formulation is breaking the continuity of the RT space,
+// e.g. we no longer require that the normal component of the RT space be
+// continuous across interelement boundaries. In the notation of Arnold and
+// Brezzi this means a change from RT_0^k -> RT_{-1}^k where k is the polynomial
+// order of the basis (note that here k denotes the polynomial order of the
+// *divergence* of the vector shape functions, so for k = 0, the RT basis in
+// libMesh is FIRST). This breakage in continuity is accomplished by changing
+// from an FEFamily of RAVIART_THOMAS to L2_RAVIART_THOMAS. Instead of being
+// enforced by the finite element space itself, continuity of the normal
+// component of the vector field is enforced through Lagrange multipliers that
+// live on the skeleton of the mesh. Let's introduce a subspace of L2, M_{-1}^k,
```

Comment:
Is "skeleton" the term they use?  If so then let's keep it here, but I'd just have said "the sides of the elements", for `SIDE_HIERARCHIC`.  "Skeleton" makes me think of simplicial complexes and CW complexes, but technically the element sides here aren't *quite* a skeleton, they're the difference between the (d-1)-skeleton (the element sides, where `SIDE_HIERARCHIC` is continuous and well-defined) and the (d-2) skeleton (the vertices and in 3D the edges, where it isn't).

Reply:
I have heard skeleton as the term used in the single hybridized discontinuous Galerkin presentation I've seen 😆